core/DataTable/Row.php

```diff
@@ -209,37 +209,9 @@ public function getColumn($name)
             return false;
         }
 
-        if ($this->isColumnValueCallable($this->c[self::COLUMNS][$name])) {
```

Comment:
Callable column values are no longer used since the processed metrics refactoring, big performance win